cadc-haproxy-dev/src/haproxy/haproxy.cfg

```diff
@@ -0,0 +1,37 @@
+global
+  #log stdout format raw local0
+  log /dev/log local0 debug
+  maxconn 256 
+  pidfile /tmp/haproxy-queue.pid
+
+defaults
+  mode http
+  timeout connect   6000ms
+  timeout client  300000ms
+  timeout server  300000ms
+  maxconn 256 
+  option redispatch
+  retries 3
+  option httpclose
+  option httplog
+  option forwardfor
+  option httpchk HEAD / HTTP/1.0
+
+frontend https-in
+   log global
+   bind *:443 ssl crt /conf/server-cert.pem ca-file /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem verify optional
+   mode http
+   option forwardfor
+   option http-server-close
+   http-request set-header X-Client-Certificate %[ssl_c_der,base64]
+   capture request header Origin len 128
```

Comment:
This CORS stuff (capture origin and beyond) has our internal domain name, making this not very reusable by someone who might want to contribute... 

Is there a sane way we could add this dynamically... get it from /conf/frontent.conf and stick it into the right part of the file?